model/licensing.py

```diff
@@ -202,6 +202,18 @@ def __repr__(self):
             self.licenses_reserved, self.patrons_in_hold_queue
         )
 
+    @classmethod
+    def exists(self, _db, identifier, collection_id, data_source_name):
```

Comment:
Defining a data model method to do a certain kind of lookup is fine, but the method should return the object looked up rather than a boolean whether or not the lookup succeeded. That way a caller has the looked-up object in case they want to do something with it.

It's also going to cause confusion to have a data model method that takes a combination of data model objects (Identifier), IDs (collection_id), and names (data_source_name).

Here's an alternate implementation that takes an Identifier, a Collection, and a DataSource -- the caller can look up those objects as necessary.

```
def for_collection(self, _db, identifier, collection, data_source):
    return get_one(_db, LicensePool, identifier=identifier, collection=collection, data_source=data_source)
```

And at this point it's a small enough amount of code that you might not need to even add a data model method.